net-tools/netwatch/src/netmon/bsd.rs

```diff
@@ -11,22 +12,24 @@ use crate::{interfaces::bsd::WireMessage, ip::is_link_local};
 
 #[derive(Debug)]
 pub(super) struct RouteMonitor {
-    handle: JoinHandle<()>,
+    #[allow(dead_code)]
+    handle: AbortOnDropHandle<()>,
```

Comment:
```suggestion
    _handle: AbortOnDropHandle<()>,
```
?

I'm not a fan of `#[allow(dead_code)]` as it easily stays around too long.  The `_` is much easier to keep track of as the compiler will tell you.

Reply:
fixed